.github/workflows/example-recording.yml

```
@@ -73,6 +73,29 @@ jobs:
           echo Cypress finished with: ${{ steps.cypress.outcome }}
           echo See results at ${{ steps.cypress.outputs.resultsUrl }}
 
+      # print the `testResults` output directly as string
+      - name: Print Cypress test results output
+        run: |
+          echo Cypress test results: ${{ steps.cypress.outputs.testResults }}
+
+      # parse the `testResults` output as JSON.
+      # Note that the output is a string and empty if there is a custom command passed to the action and the command runs through CLI
+      - name: Parse the Cypress test results output
+        uses: actions/github-script@v6.4.1
```

```suggestion
        uses: actions/github-script@v6
```

Use the major version `@v6`

Unfortunately this is not yet compatible with `node20` so this will generate deprecation warnings starting Oct 23, 2023. This is not so serious, since it will only show in one of the examples.
- Also there is a PR open https://github.com/actions/github-script/issues/421 to update to `node20`